app/controllers/concerns/navigation.rb

```diff
@@ -11,7 +11,7 @@ module Navigation
   attr_reader :active_categories
 
   def assign_active_categories(*categories)
-    categories.each do |category|
+    categories.map do |category|
```

Comment:
That's what it looks like, but I couldn't see it being assigned or used anywhere like that.